sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -87,23 +93,40 @@ export class AssessmentAESTController extends BaseController {
    */
   @Get("application/:applicationId/history")
   async getAssessmentHistorySummary(
```

Comment:
Is this method not supposed to support pagination ?

Reply:
no @henryoforeh-aot in a real scenario, the history and requested summary will have only a few rows, so there is no need for lazy loading

Reply:
Okay thanks for providing some more info on it. I will resolve.